dask_cudf/io/csv.py

```diff
@@ -47,8 +64,30 @@ def _read_csv(fn, dtypes=None, **kwargs):
     except GDFError:
         # end of file check https://github.com/rapidsai/dask-cudf/issues/103
         # this should be removed when CUDF has better dtype/parse_date support
-        dtypes = dict(zip(kwargs['names'], dtypes))
+        dtypes = dict(zip(kwargs["names"], dtypes))
         df = dd.core.make_meta(dtypes)
         cdf = cudf.from_pandas(df)
+    return cdf
+
+
+def read_csv_with_compression(path, **kwargs):
```

Comment:
Maybe rename this to `read_csv_without_chunksize`

Alternatively, maybe we remove this function and inline the contents.